chains/metertestnet/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://testnet.meter.io
```

Comment:
is there an api for verifying contracts?

Reply:
From what I've found, unfortunately, Meter (mainnet and testnet) uses Sourcify for contract verification and there is no clear api endpoint for that; It looks like it's done through web interfaces (Sourcify portal/Meter explorer..)